scripts/globals/items/scroll_of_instant_retrace.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------------
+-- ID: 5428
+-- Scroll of Instant Retrace
+-- Transports the user to their Allied Nation.
+-----------------------------------------
+require("scripts/globals/teleports")
+require("scripts/globals/status")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if (target:getCampaignAllegiance() > 0) then
+        result = 0
+    else
+        result = 56
+    end
+    return result
+end
+
+function onItemUse(target)
+    if (target:getCampaignAllegiance() > 0) then
+        target:addStatusEffectEx(tpz.effect.TELEPORT, 0, tpz.teleport.id.RETRACE, 0, 1)
```

Comment:
1 second? seems kinda short for that animation. But if you already checked and its retail accurate leave it.

Reply:
I was using the Scroll of Instant Warp item as a base line reference, but 1 did feel too short for this.